.circleci/config.yml

```diff
@@ -0,0 +1,78 @@
+version: 2.1
+
+orbs:
+  android: circleci/android@2.5.0
+
+commands:
+  install-android-sdk-on-macos:
+    description: Install the Android SDK on macOS
+    steps:
+      - run:
+          name: Install Android SDK commandline tools
+          command: brew install --cask android-commandlinetools
+      - run:
+          name: Set Android SDK environment variables
+          command: |
+            echo 'export ANDROID_HOME="$HOMEBREW_PREFIX/share/android-commandlinetools"' >> "$BASH_ENV"
+            echo 'export PATH="$ANDROID_HOME/emulator:$ANDROID_HOME/tools:$ANDROID_HOME/tools/bin:$ANDROID_HOME/platform-tools:$PATH"' >> "$BASH_ENV"
+      - android/accept-licenses
+      - run:
+          name: Install Android SDK platform tools
+          command: sdkmanager "platform-tools"
+  require-snapshot-version:
+    description: Check that the current version is a SNAPSHOT version
+    steps:
+      - run:
+          name: Check that the current version is a SNAPSHOT version
+          command: |
+            set -euo pipefail
+            file_path="gradle/libs.versions.toml"
+            version=$(grep 'revenuecat-kmp = ' "$file_path" | awk -F ' = ' '{print $2}' | tr -d '"')
+            if [[ ! "$version" =~ -SNAPSHOT$ ]]; then
+              echo "$version is not a SNAPSHOT version. Exiting..."
+              exit 1
+            else
+              echo "$version is a SNAPSHOT version. Proceeding..."
+            fi
```

Comment:
not needed for now, but this should be moved to fastlane because it's easier to maintain and reuse. It's easier to use ruby than bash

Reply:
Yes agreed! Although for now I don't even know how permanent this behavior is going to be, but will definitely move it to fastlane if it stays.